javascript/javascript1/week2/studman.js

```diff
@@ -0,0 +1,23 @@
+const class07Students = [];
+const maxStudents = 6;
```

Comment:
Very nice use of constant to store this value! 🌟 As a brief thought experiment, can you explain why this is good and how it helps you and others maintain code? 😄

Reply:
I think that storing the maximum number of students into one variable makes it easier to find in the code (with the name😂) and make changes if necessary 🤔